mat64/solve.go

```diff
@@ -0,0 +1,87 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mat64
+
+import (
+	"github.com/gonum/blas"
+	"github.com/gonum/lapack/lapack64"
+)
+
+// Solve solves a minimum-norm solution to a system of linear equations defined
+// by the matrices a and b. If a is singular or near-singular a Condition error
```

Comment:
Oops. Fixed to return Condition. I'll remove ErrSingular in the next PR, as it's still referenced in documentation.